appserver/tests/paas/basic-jpa/pom.xml

```diff
@@ -93,7 +93,7 @@
             <scope>test</scope>
         </dependency>
         <dependency>
-      	    <groupId>javax.persistence</groupId>
+      	    <groupId>jakarta.persistence</groupId>
             <artifactId>persistence-api</artifactId>
```

Comment:
I think this one doesn't exist, see:

https://mvnrepository.com/artifact/jakarta.persistence

Reply:
yes it does

Reply:
Are you sure? Looking at Maven central directly, I only see a folder called "jakarta.persistence-api":

https://repo1.maven.org/maven2/jakarta/persistence/

Maven also gives the following error:

> Failure to find jakarta.persistence:persistence-api:jar:2.2.2 in https://repo.maven.apache.org/maven2 was cached in the local repository, resolution will not be reattempted until the update interval of central has elapsed or updates are forced -> [Help 1]

Reply:
I also checked only `jakarta.persistence-api` artifacts exist.